src/api/Config.json

```diff
@@ -1,119 +1,121 @@
+        "name": "traction",
+        "apiNamespace": "v1",
+        "baseURL": "VUE_APP_TRACTION_BASE_URL",
         "resources": [
-          {
-            "name": "libraries"
-          },
-          {
-            "name": "enzymes"
-          },
-          {
-            "name": "runs",
-            "include": "chip.flowcells.library"
-          },
-          {
-            "name": "chips"
-          },
-          {
-            "name": "flowcells"
-          },
-          {
-            "name": "requests",
-            "include": ""
-          },
-          {
-            "name": "tubes",
-            "include": "materials"
-          }
+            {
+                "name": "samples"
+            },
+            {
+                "name": "tags"
+            }
+        ],
+        "pipelines": [
+            {
+                "name": "saphyr",
+                "resources": [
+                    {
+                        "name": "libraries"
+                    },
+                    {
+                        "name": "enzymes"
+                    },
+                    {
+                        "name": "runs",
+                        "include": "chip.flowcells.library"
+                    },
+                    {
+                        "name": "chips"
+                    },
+                    {
+                        "name": "flowcells"
+                    },
+                    {
+                        "name": "requests",
+                        "include": ""
+                    },
+                    {
+                        "name": "tubes",
+                        "include": "materials"
+                    }
+                ]
+            },
+            {
+                "name": "pacbio",
+                "resources": [
+                    {
+                        "name": "libraries",
+                        "include": "requests"
+                    },
+                    {
+                        "name": "request_library",
+                        "include": ""
+                    },
+                    {
+                        "name": "requests",
+                        "include": ""
+                    },
+                    {
+                        "name": "tubes",
+                        "include": "materials"
+                    },
+                    {
+                        "name": "runs",
+                        "include": "plate.wells.libraries",
+                        "resources": [
+                            {
+                                "name": "plates",
+                                "include": ""
+                            },
+                            {
+                                "name": "wells",
+                                "include": ""
+                            }
+                        ]
+                    },
+                    {
+                        "name": "plates",
+                        "include": "plates.wells.materials"
```

Comment:
this can be just include:"wells.materials" but that change is also in my PR so ill just wait til merge and resolve the conflict